### PR TITLE
Added possibility to enable/disable an animation

### PR DESCRIPTION
**Pull request by DrunkenDealer**

### CODE REVIEW

File: gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt [hunk 1]
```diff
@@ -272,7 +274,11 @@ open class GaugeProgressView : View {
 
     fun setProgress(newProgress: Int) {
```
Comment (edgar-zigis) [inline gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt:275]: better supply animated parameter to this method with a default value true.

**Review by edgar-zigis — CHANGES_REQUESTED** [review]

File: gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt (hunk 1 above)
Comment (DrunkenDealer) [reply gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt]: Done

File: gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt [hunk 2]
```diff
@@ -270,9 +270,13 @@ open class GaugeProgressView : View {
 
     //  Public setters
 
-    fun setProgress(newProgress: Int) {
+    fun setProgress(newProgress: Int, animated: Boolean = true) {
         val targetProgress = max(0, min(newProgress, 100))
-        val duration = (abs(progress - newProgress).toFloat() / 10) * 200
+        var duration = (abs(progress - newProgress).toFloat() / 10) * 200
```
Comment (edgar-zigis) [inline gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt:275]:
```
val duration = if (animated) {
    (abs(progress - newProgress).toFloat() / 10) * 200
} else {
    0f
}
```

**Review by edgar-zigis — CHANGES_REQUESTED** [review]

File: gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt (hunk 2 above)
Comment (DrunkenDealer) [reply gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt]: Done

**Review by edgar-zigis — APPROVED** [review]

**Pull request merged**